src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1018,7 +1018,7 @@ void GraphOptimizer::FuseConvolutionAndZeroPoints(Graph& graph) {
             OPENVINO_THROW("Cannot cast to TBlob internal weights blob");
         }
 
-        auto weightsPtr = static_cast<const int8_t*>(weightsBlob->getData());
+        auto weightsPtr = static_cast<const int8_t*>(weightsBlob->getData());  // NOLINT(bugprone-signed-char-misuse)
```

Comment:
Is it actually an incorrect warning?
I see that later this pointer is addressed as int32_t and never used as int8_t

Reply:
Can we use CharTypdefsToIgnore

Reply:
Done